models/Settings.php

```diff
@@ -10,11 +10,24 @@ class Settings extends Model
 
     use \October\Rain\Database\Traits\Validation;
     public $rules = [
-        'digits'                => 'required_if:is_enabled,1,true',
+        'digits' => 'required_if:is_enabled,1,true|numeric',
     ];
     // A unique code
     public $settingsCode = 'vannut_wipprotect_settings';
 
     // Reference to field configuration
     public $settingsFields = 'fields.yaml';
+
+    public function afterValidate()
```

Comment:
I'm not exactly sure if I should have used `beforeValidate` or `afterValidate` here. Both seem to work the same. I believe on is to do a custom validation before what is found in the `$rules` property is executed and the other is after. As long a an exception is returned, the data will be rejected either way.

Reply:
Thanks @osmanzeki for the PRs!! 

AFAIK you don’t need to use an onBefore or onAfterValidate just to validate the user input. You have to initiate it in the yaml of the model. Ill look into it asap; and review your other PRs asap

Reply:
I tried using the "numeric" validation rule from Laravel, but it accepts hexadecimal values. If I don't do the afterValidate, the form does not show any error to the user but the password is also not saved (the value 0 is saved instead).